lib/Controller/RowController.php

```diff
@@ -64,13 +66,19 @@ public function show(int $id): DataResponse {
 	public function create(
 		?int $tableId,
 		?int $viewId,
-		array $data
-	): DataResponse {
-		return $this->handleError(function () use ($tableId, $viewId, $data) {
-			return $this->service->create(
+		array $data,
+		SuperService $superService
+	): DataResponse { 
+		return $this->handleError(function () use ($tableId, $viewId, $data, $superService) { 
+
+			$create = $this->service->create(
```

Comment:
For some more obvious naming

```suggestion
			$newRow = $this->service->create(
```